### DISCUSSION TITLE
How many socks must he take out to be 100% sure he is holding at least one pair of black socks?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#415] [Galaxy Trivia] [answered]
A man has 53 socks in his drawer: 21 identical blue, 15 identical black and 17 identical red. The lights are out and he is completely in the dark.

**Comment by mayankjaiswal1709** [comment, marked as answer]
To answer such questions we always have to assume that the worst case scenario has occurred, since if the worst case scenario is covered all other scenarios have definitely been covered.

So in this case lets assume:

The first 21 socks he takes out are all blue

The next 17 socks he takes out are all red

Since now only black socks remain in the drawer the next 2 socks he will draw will surely be black.

So, total number of minimum socks he has to take out to be absolutely sure that he has a pair of black socks is 21+17+2 = 40

**Reply by ParanoidUser** [reply]
Bingo! One more to go for the Galaxy Brain badge. 😊

**Reply by peterbaccaro** [reply]
> Bingo! One more to go for the Galaxy Brain badge. 😊

That's the wrong answer as the correct answer to the puzzle is zero because the man already knew he had black socks before the lights went out so when the lights went out he didn't need to take any socks out of the draw to be 100% sure that he had at least one pair.  It's a trick question in the way it's written and doesn't say that the goal is to be holding a pair of black socks the goal is simply to be 100% sure that he has them.

**Reply by ParanoidUser** [reply]
That's a pretty general problem statement written in schoolbooks. I agree that it's very hard to believe in the sequence of events:
 1. a man doesn't know where his socks are (simply impossible 😂);
 2. and he desperately needs a black pair;
 3. and out of the blue, the lights go out.

Any suggestions for improving the problem description are greatly appreciated.

**Reply by peterbaccaro** [reply]
The question needs to say, how many socks does he need to take out to be 100% sure he **is holding** at least one pair of black socks.  

Otherwise the question is simply asking what he needs to do in order to be 100% sure that he **has** one pair of black socks however the question has already stated that he has multiple pairs of black socks in the draw so he doesn't need to do anything to be 100% sure.

**Reply by ParanoidUser** [reply]
**Edited.** Thank you for your help!

**Comment by peterbaccaro** [comment]
**Zero** socks as the question does not state that he needs to have them in his hands and question has stated that he has multiple pairs in his draw.
